db/migrations/20230320085600-add-commit-from-all-entities.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn("RepoSyncStates", "branchFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "pullFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "buildFrom", { type: Sequelize.DATE, allowNull: true });
+    await queryInterface.addColumn("RepoSyncStates", "deploymentFrom", { type: Sequelize.DATE, allowNull: true });
```

Comment:
What about `issueFrom`? is it skipped intentionally?

Reply:
I looked at that, didn't think we have an task for issue? Although we have cols.

Reply:
We can remove the `issue` column if it is obsolete one; may be in a seperate ticket.

Reply:
Yup. Definitely in another PR.